pkg/security/admission/admission_test.go

```diff
@@ -1154,3 +1116,24 @@ func goodPod() *kapi.Pod {
 		},
 	}
 }
+
+func containerSC(uid int64) *kapi.SecurityContext {
+	no := false
+	return &kapi.SecurityContext{
+		Privileged: &no,
+		RunAsUser:  &uid,
+		SELinuxOptions: &kapi.SELinuxOptions{
+			Level: "s0:c1,c0",
+		},
+	}
+}
+
+func podSC(fsGroup, supGroup int64) *kapi.PodSecurityContext {
+	return &kapi.PodSecurityContext{
+		SELinuxOptions: &kapi.SELinuxOptions{
+			Level: "s0:c1,c0",
```

Comment:
I see that this is the way it was in the previous tests but now I'm wondering why it's hard coded.

Reply:
I had a version where SELinux Level was passed to the function, but later I realized that we always passing the same value and I decided to inline it. With the current approach we need to pass only the values that differs between tests.

Do you think that I should step back and continue to passing a level as an argument?

Reply:
The value assumption is correct but we should be pulling it from the namespace.  Guess that was missed in a refactor somewhere:

```
func CreateNamespaceForTest() *kapi.Namespace {
	return &kapi.Namespace{
		ObjectMeta: metav1.ObjectMeta{
			Name: "default",
			Annotations: map[string]string{
				allocator.UIDRangeAnnotation:           "1/3",
				allocator.MCSAnnotation:                "s0:c1,c0",
				allocator.SupplementalGroupsAnnotation: "2/3",
			},
		},
	}
}
```

Reply:
I updated the code and started to use a variable for that. PTAL.